example/CMakeLists.txt

```diff
@@ -32,6 +32,23 @@ if(BUILD_TESTING)
   set(ament_cmake_flake8_FOUND TRUE) # Conflicts with black
   set(ament_cmake_uncrustify_FOUND TRUE) # Conflicts with clang-format
   ament_lint_auto_find_test_dependencies()
+
+  find_package(generate_parameter_library REQUIRED)
+  find_package(rclcpp REQUIRED)
```

Comment:
These are not needed as they are find_pacakged on lines 5 and 7 of this file.

Reply:
I am also used to using them in the test-defintion part to increase clarity of the file. Nevertheless we can remove it.